packages/core/src/Serialization/Certificates/CertificateKind.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Certificates are used to register, update, or deregister stake pools, and delegate stake.
```

Comment:
I had a question why not consolidate this with `Cardano.CertificateType`, so probably worth adding a comment to explain
```suggestion
 * Certificates are used to register, update, or deregister stake pools, and delegate stake.
 * Values are used for serialization.
```